ios/swift/shared/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -97,6 +97,11 @@
       "idiom" : "ipad",
       "filename" : "logo1024-ipad-83@2x.png",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ios-marketing",
+      "size" : "1024x1024",
+      "scale" : "1x"
```

Comment:
revert pls. I'll organize the image folder in a separate PR.

Reply:
done